tests/foreman/ui/test_location.py

```diff
@@ -770,34 +800,38 @@ def test_remove_configtemplate(self, testdata):
         @assert: config template is added and then removed
 
         """
-        bug_id = testdata.pop('bugzilla', None)
-        if bug_id is not None and bz_bug_is_open(bug_id):
-            self.skipTest('Bugzilla bug {0} is open.'.format(bug_id))
-
-        template = testdata['user_name']
         strategy, value = common_locators['all_values_selection']
-        loc_name = gen_string('alpha')
         with Session(self.browser) as session:
-            set_context(session, org=ANY_CONTEXT['org'])
-            make_templates(
-                session,
-                name=template,
-                template_path=get_data_file(OS_TEMPLATE_DATA_FILE),
-                template_type='provision',
-                custom_really=True,
-            )
-            self.assertIsNotNone(self.template.search(template))
-            make_loc(session, name=loc_name)
-            self.location.search(loc_name).click()
-            session.nav.click(tab_locators['context.tab_template'])
-            element = session.nav.wait_until_element(
-                (strategy, value % template))
-            # Item is listed in 'Selected Items' list and not 'All Items' list.
-            self.assertIsNotNone(element)
-            self.template.delete(template)
-            self.location.search(loc_name).click()
-            session.nav.click(tab_locators['context.tab_template'])
-            element = session.nav.wait_until_element(
-                (strategy, value % template))
-            # Item is listed in 'All Items' list and not 'Selected Items' list.
-            self.assertIsNone(element)
+            for testdata in valid_config_template_names():
```

Comment:
It seems that BZ 1164247 was closed a long time ago and is not actual anymore, so I propose to remove `valid_config_template_names()` method and that BZ part:
```
bug_id = testdata.pop('bugzilla', None)
if bug_id is not None and bz_bug_is_open(bug_id):
    self.skipTest(
          'Bugzilla bug {0} is open.'.format(bug_id))
```